client/src/App.js

```diff
@@ -1,27 +1,14 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
 import "./App.css";
 
-import Hello from "./components/hello";
+import RepoViews from "./views/RepoViews";
```

Comment:
I feel like, even if it's not subject to this change, that the css should be imported first maybe? Usually, when I modify a component, I almost never modify the css file, but I do add other imports. And then, the order of imports breaks.

Reply:
Updated.